synfig-studio/plugins/lottie-exporter/effects/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+set(INSTALL_DESTINATION "share/synfig/plugins/lottie-exporter/effects")
+set(PLUGIN_LOTTIE_EXPORTER_EFFECTS_FILES
+    allmask.py
```

Comment:
Maybe group lottie plugin files to something like:
```
set(PLUGIN_LOTTIE_EXPORTER
    effects/allmask.py
    ...
    helpers/bezier.py
```
and so on?

Reply:
Won't the way it's done be more consistent? Even other projects follow something similar where each directory has it's own *CMakeLists.txt* file.

Reply:
The main idea is to make the code cleaner and more understandable. But when I look at this code, I can’t understand what it is trying to do. Is there any logic inside subfolders? Does it change some files? Also this code will be difficult to maintain.

One of the reasons to creating makefiles in each directory is just a comfortable way to run the `make` command without going to the parent directory. But here we are not compiling anything, just copying the directory to some place.

And about other projects. Here is a piece of Inkscape code, for example::
https://gitlab.com/inkscape/inkscape/-/blob/master/src/trace/CMakeLists.txt